### PR TITLE
Account pages i18n

### PR DESCRIPTION
**Pull request by philcon93**
- Toggled 'Zip code/ postal code' label,
- Made required `*` consistent on account pages,
- Only show ABN field for Australian merchants.

### CODE REVIEW

File: src/templates/customer/edit_account/template.html [hunk 1]
```diff
@@ -173,7 +173,7 @@ <h1>Edit Your [@config:company_name@] Account Details</h1>
 				}
 
 				if(count <= 0) {
-					msg.innerHTML = '<span class="text-danger">Invalid Post Code.</span>';
+					msg.innerHTML = '<span class="text-danger">Invalid Postal Code.</span>';
```
Comment (jeff-stamborski) [inline src/templates/customer/edit_account/template.html]: Why not do the same 
`[%if [@config:defaultcountry@] eq 'US'%]Zip [%else%]Postal [%/if%]`
here?

File: src/templates/customer/edit_account/template.html [hunk 2]
```diff
@@ -268,45 +261,45 @@ <h2 name="billing" id="billing">Billing Information</h2>
...
 					<input class="form-control" type="text" name="phone" id="phone" size="50" value="[@phone@]">
 				</div>
 				<div class="form-group">
-					<label for="fax">Fax Number</label>
+					<label for="fax">Fax number</label>
 					<input class="form-control" type="text" name="fax" id="fax" size="50" value="[@fax@]">
 				</div>
 				<!-- Company Information -->
 				<h2 name="company" id="Company">Company Information</h2>
+				[%if [@config:defaultcountry@] eq 'AU'%]
```
Comment (jeff-stamborski) [inline src/templates/customer/edit_account/template.html]: As discussed these ABN configs are kinda of a small mess.

Perhaps we change this logic to 
`If defaultcountry == 'AU' || primary_abn_label != 'ABN'`
??? 😩

File: src/templates/customer/edit_account/template.html [hunk 3]
```diff
@@ -268,45 +261,45 @@ <h2 name="billing" id="billing">Billing Information</h2>
...
 					<!-- Dropshipping Information -->
 					<h2>Dropshipping Information</h2>
-					<h4>Customise the documentation we send out with your drop shipped orders.</h4>
+					<b>Customise the documentation we send out with your drop shipped orders.</b>
 					<label>Drop Shipping Documentation Footer (Will be displayed on footer of packing slips, labels etc)</label>
 					<textarea class="form-control"  name="usercustom7" cols="50">[@usercustom7@]</textarea>
 					<p class="help-block">(Instructions: Enter the text you would like to see on the footer of any documentation we send out to your customers. EG: Call 1300 123 567 to re-order. When you are finished click the &quot;save changes I have made&quot; button at the bottom of this page.)</p>
 					<hr>
-					<label>Dropshipping Company Logo (Will be displayed on header of packing slips, labels etc)</label>
+					<label>Dropshipping company logo (Will be displayed on header of packing slips, labels etc)</label>
```
Comment (jeff-stamborski) [inline src/templates/customer/edit_account/template.html]: I feel like "Will" should be "will". No capital in parenthesis.

File: src/templates/customer/edit_address/template.html [hunk 4]
```diff
@@ -242,7 +232,7 @@ <h2>Address Details</h2>
 			}
 		}
 		if(count <= 0) {
-			msg.innerHTML = '<span class="text-danger">Invalid Post Code.</span>';
+			msg.innerHTML = '<span class="text-danger">Invalid Postal Code.</span>';
```
Comment (jeff-stamborski) [inline src/templates/customer/edit_address/template.html]: Same as above. Why not have the (if USA then "Zip") logic?

File: src/templates/customer/wholesaleregister/template.html [hunk 5]
```diff
@@ -102,14 +102,16 @@ <h3>Company Information</h3>
 					</select>
 				</div>
 			</div>
+			[%if [@config:defaultcountry@] eq 'AU'%]
```
Comment (jeff-stamborski) [inline src/templates/customer/wholesaleregister/template.html]: Same as other file. ABN nonsense.

File: src/templates/customer/edit_account/template.html [hunk 6]
```diff
@@ -268,45 +261,45 @@ <h2 name="billing" id="billing">Billing Information</h2>
...
 					<input class="form-control" type="text" name="phone" id="phone" size="50" value="[@phone@]">
 				</div>
 				<div class="form-group">
-					<label for="fax">Fax Number</label>
+					<label for="fax">Fax number</label>
 					<input class="form-control" type="text" name="fax" id="fax" size="50" value="[@fax@]">
 				</div>
 				<!-- Company Information -->
 				<h2 name="company" id="Company">Company Information</h2>
+				[%if [@config:defaultcountry@] eq 'AU' || [@config:PRIMARY_ABN_LABEL@] ne 'ABN'%]
```
Comment (jeff-stamborski) [inline src/templates/customer/edit_account/template.html]: Does the || operator work in BASE [%if%]? 

I thought it needed to be **or**...

File: src/templates/customer/wholesaleregister/template.html [hunk 7]
```diff
@@ -102,14 +102,16 @@ <h3>Company Information</h3>
 					</select>
 				</div>
 			</div>
+			[%if [@config:defaultcountry@] eq 'AU' || [@config:PRIMARY_ABN_LABEL@] ne 'ABN'%]
```
Comment (jeff-stamborski) [inline src/templates/customer/wholesaleregister/template.html]: Same as above question.

File: src/templates/customer/edit_account/template.html [hunk 8]
```diff
@@ -173,7 +173,7 @@ <h1>Edit Your [@config:company_name@] Account Details</h1>
 				}
 
 				if(count <= 0) {
-					msg.innerHTML = '<span class="text-danger">Invalid Post Code.</span>';
+					msg.innerHTML = '<span class="text-danger">Invalid [%if [@config:defaultcountry@] eq 'US'%]Zip [%else%]Postal [%/if%] Code.</span>';
```
Comment (neto-ben) [inline src/templates/customer/edit_account/template.html]: why are these capitals but line 89 is not?

File: src/templates/customer/edit_account/template.html [hunk 9]
```diff
@@ -268,45 +261,45 @@ <h2 name="billing" id="billing">Billing Information</h2>
...
+					<label for="usercustom3">Web site url</label>
 					<input class="form-control"  type="text" name="usercustom3" id="usercustom3" size="100" maxlength="255" value="[@usercustom3@]">
 				</div>
 				[%CONFIG id:'ALLOW_DROPSHIP_CHECKOUT' if:'=' value:'1'%]
 					<!-- Dropshipping Information -->
 					<h2>Dropshipping Information</h2>
-					<h4>Customise the documentation we send out with your drop shipped orders.</h4>
-					<label>Drop Shipping Documentation Footer (Will be displayed on footer of packing slips, labels etc)</label>
+					<b>Customise the documentation we send out with your drop shipped orders.</b>
+					<label>Drop Shipping Documentation Footer (will be displayed on footer of packing slips, labels etc)</label>
```
Comment (neto-ben) [inline src/templates/customer/edit_account/template.html]: missed the title case here :)

File: src/templates/customer/edit_account/template.html [hunk 10]
```diff
@@ -268,45 +261,45 @@ <h2 name="billing" id="billing">Billing Information</h2>
...
 				<div class="form-group">
-					<label for="usercustom3">Your Web Site URL</label>
+					<label for="usercustom3">Web site url</label>
 					<input class="form-control"  type="text" name="usercustom3" id="usercustom3" size="100" maxlength="255" value="[@usercustom3@]">
 				</div>
 				[%CONFIG id:'ALLOW_DROPSHIP_CHECKOUT' if:'=' value:'1'%]
 					<!-- Dropshipping Information -->
 					<h2>Dropshipping Information</h2>
-					<h4>Customise the documentation we send out with your drop shipped orders.</h4>
+					<b>Customise the documentation we send out with your drop shipped orders.</b>
```
[the remark this answers is not in the document]
Comment (neto-ben) [reply src/templates/customer/edit_account/template.html]: Maybe we need a `l18n` tag or something? `[%l18n%]Customise[%/l18n%]` would map that word to `Customize` when default country is US

[the remark this answers is not in the document]
Comment (neto-ben) [reply src/templates/customer/edit_account/template.html]: OK I have been informed of https://netoecommerce.atlassian.net/browse/CAT-286 which is slightly more elegant.

**Pull request merged**